msteams-platform/concepts/build-and-test/share-in-teams.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Share-in-Teams
+description: Learn to add the Share in Teams embedded on your personal app or tab
+ms.topic: reference
+ms.localizationpriority: medium
+keywords: Share Teams Share-in-Teams
+---
+# Create Share-in-Teams
+
+> [!NOTE]
+> Share-in-Teams is currently available only in [public developer preview](../../resources/dev-preview/developer-preview-intro.md).
+
+Share-in-Teams allows you to share the content from personal app or tab to another user within Teams. You can select Share-in-Teams to launch the Share-in-Teams experience in a pop-up window. This pop-up window allows you to add another user or group or channel, notes and share the content.
+
+The following image shows the Share-in-Teams pop-up window:
+
+:::image type="content" source="../../assets/images/share-in-teams/share-in-teams.PNG" alt-text="share-in-teams":::
+
+## Enable Share-in-Teams
+
+> [!NOTE]
+> Ensure that you have [Microsoft Teams JavaScript Client SDK v2 Preview](/javascript/api/overview/msteams-client?view=msteams-client-js-beta&preserve-view=true&branch=pr-en-us-5129) (`@microsoft/teams-js@1.11.0-beta.7`) to enable Share-in-Teams.
+
+To enable Share-in-Teams in your personal tab or app,
+call `microsoftTeams.sharing.shareWebContent` with a payload.
+
+The following example explains how to create a payload:
+
+```json
+microsoftTeams.sharing.shareWebContent({
+        content: [
+          {
+            type: 'URL',
+            url: 'URL which the dev wants to be shared',
+            preview: true
+          }
+        ]
+      });
+```
+
+The payload contains the following parameters:
+
+| Property name | Purpose |
+|---|---|
+| `type` | The type must be `URL` |
+| `url` | Required URL |
+| `preview` | Preview must be `true` |
+
+The following image shows the Share-in-Teams option:
+
+:::image type="content" source="../../assets/images/share-in-teams/share-button.PNG" alt-text="share-in-teams-button":::
+
+## End user Share-in-Teams experience
+
+1. Open a personal app or tab and select a content to share.
+
+2. From the ellipsis (...) menu, select **Share-to-Teams**.
```

Comment:
This is irrelevant. The app developer customizes the trigger for how sharing will happen. It could be a button or a menu or a link.

Reply:
Helen is correct here.